libs/sdk-ui-kit/src/FilterLabel/FilterLabel.tsx

```diff
@@ -0,0 +1,124 @@
+// (C) 2007-2020 GoodData Corporation
+import React, { ReactNode } from "react";
+import { injectIntl, WrappedComponentProps } from "react-intl";
+
+import { IFilterLabelProps, IFilterLabelState } from "./typings";
+
+class WrappedFilterLabel extends React.PureComponent<
+    IFilterLabelProps & WrappedComponentProps,
+    IFilterLabelState
+> {
+    static defaultProps: Partial<IFilterLabelProps & WrappedComponentProps> = {
+        isAllSelected: false,
+        isDate: false,
+        selection: "",
+        selectionSize: null,
+        noData: false,
+    };
+    private labelRef: HTMLSpanElement;
+
+    constructor(props: IFilterLabelProps & WrappedComponentProps) {
+        super(props);
+
+        this.state = { hasEllipsis: false };
+        this.setLabelRef = this.setLabelRef.bind(this);
```

Comment:
Can we avoid the bind? Or better yet, use createRef?

Reply:
Fixed by `createRef`.